April CQA work for MTA 8.1 Release Notes DITA fixes (#356)
migtools/mta-documentation: docs/topics/release-notes-topics/ref_fixed-issues-8-1-0.adoc

= Fixed issues

[role="_abstract"]

> Review the following fixed issues that have a significant impact in {ProductFirstRef} ({ProductShortName}) version 8.1.0.

Reviewer: ```suggestion
Review issues that have been fixed in {ProductFirstRef} ({ProductShortName}) version 8.1.0.
```

Creating Source Control credentials no longer fails when the password exceeds the character limit::

> Before this update, you could not create Source Control credentials in the {ProductShortName} user interface (UI) if you entered a user credential password that exceeded the 120-character limit. With this update, {ProductShortName} does not the limit for maximum length for user credentials. As a result, you can create Source Control credentials with longer passwords.

Reviewer: ```suggestion
Before this update, you could not create Source Control credentials in the {ProductShortName} user interface (UI) if you entered a user credential password that exceeded the 120-character limit. With this update, {ProductShortName} does not limit the maximum length for user credentials. As a result, you can create Source Control credentials with longer passwords.
```

Creating Source Control credentials no longer fails when the password exceeds the character limit::

> Before this update, you could not create Source Control credentials in the {ProductShortName} user interface (UI) if you entered a user credential password that exceeded the 120-character limit. With this update, {ProductShortName} removes the limit for maximum length for user credentials. As a result, you can create Source Control credentials with longer passwords.

Reviewer 2: "{ProductShortName} removes the limit" - anthropomorphism. Fix if it occurs elsewhere too.
